Tests/syntax_test_comparison_operators.pb

```diff
@@ -0,0 +1,41 @@
+﻿;; SYNTAX TEST "Packages/Purebasic/Purebasic.sublime-syntax"
```

Comment:
I checked out the branch and this whole test file fails to load the referenced syntax definition because of the wrong capitalization you mentioned in the related issue. When this is correct all assertions are confirmed successfully.

Reply:
I've already fixed that, but the GHAction doesn't restart when I force push, so it's not detecting the fix.

Reply:
> You retained the .purebasic suffix in the syntax definition but dropped it in the assertions. Was that on purpose and if so, then why?

These are the best practices guidelines from the official community guide on syntaxes and syntax tests. Beside behind redundant, since the scope being tested is always only `source.purbasic`, it would also break tests in case in the future we add some further scope specifiers (e.g. `operator.comparison.gt`, etc.).

Reply:
I checked out your branch directly from your fork and it was not fixed.

According to [the official documentation](https://www.sublimetext.com/docs/scope_naming.html#syntax_definitions) the syntax name is recommended to be the last part of scope names. Hence I did it that way. Also, because all examples I have seen so far reflect that. On the other hand: you are right. They can be omitted.

Reply:
> I checked out your branch directly from your fork and it was not fixed.

It was fixed on the PR branch as soon as I realized the problem (locally it runs fins, since Win OS has a case-insensitive files system), as you can see from the [changed files diff].

But the GitHub Actions Workflow is still reporting it in wrong casing.
If you look at the error reported at [line 12]:

```
/home/runner/work/purebasic-language-for-sublime-text/purebasic-language-for-sublime-text/Data/Packages/PureBasic/Tests/syntax_test_comparison_operators.pb:1 Error loading syntax file "Packages/Purebasic/Purebasic.sublime-syntax"
```

It still mentions the old letter casing mistake `Purebasic.sublime-syntax`.

I don't know why the job doesn't restart automatically when it detects a forced-push, but the error log is outdated. Either it lacks refresh settings, or it has wrong cache settings.

Can't you restart it manually?

<!----------------------------- REFERENCE LINKS ------------------------------>

[changed files diff]: https://github.com/peterthomashorn/purebasic-language-for-sublime-text/pull/47/files#diff-e82e01474ab0fd39837befc7345d9ec959d96fe776614b08f596a49c76de234dR1

[line 12]: https://github.com/peterthomashorn/purebasic-language-for-sublime-text/runs/4959927673?check_suite_focus=true#step:6:12

<!-- EOF -->

Reply:
> There are options for case-sensitive file systems

Yes, now Windows 10 and above also have that option, on a per folder basis, but requires activating WSL. I haven't found the time to test it, but it's indeed a good feature for working with cross-platform repositories (Git or otherwise).

But I'm not sure if it can be enabled in system folders, including the ST packages folder which in Win OS is under `AppData/` — these sys folders might either be excluded from the feature, or unwise to fiddle with.

> I am not surprised the CI runner fails, I guess it uses ext4 or something else.

Surely, the CI will use a Unix like file system, but the problem is that the workflow doesn't restart on force pushes. You should look into it and find out why. On Travis CI, builds are automatically re-run when the PR branch changes (force pushes or otherwise); this is the default behaviour, which can be tweaked in the options (along with caching, etc.). The problem with these CI systems is that you end up reading docs for hours, and they also change quite often.